mon-pix/app/components/pix-action-chip.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+export default class PixActionChip extends Component {
+  @tracked
+  isTriggering = false;
+
+  get actionTitle() {
+    return this.args.actionTitle;
+  }
+
+  get actionCompleted() {
+    return this.args.actionCompleted;
+  }
+
+  get isTriggering() {
+    return this.isTriggering;
+  }
```

Comment:
Simple question, est-ce utile d'avoir un getter lorsque la donnée n'y est pas modifiée ?

Reply:
Non, ce n'est pas utile et je ne sais même pas comment Ember fait la différence entre le `isTriggering` tracké et le getter lors d'un appel à `this.isTriggering`.

Reply:
Merci